### PR TITLE
chore: update RN from v0.63 to v0.64

### PR DESCRIPTION
**Pull request by patogallaiovlabs**
## Description

From previous steps 0.59->0.63 (#690 / #698 / #702 / #703), this step goes from 0.63 to 0.64.
Changelog of 0.64: https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#v0640
We followed this guide:     
- https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2

## Known issue
Parse lib has a known issue, we fix it with a patch:
- https://github.com/microsoft/react-native-windows/issues/7451#issuecomment-856392511

## How Has This Been Tested?

Doing some basics interactions, txs send, etc.

## Types of changes

- [ ] fix: bug fixes, e.g. fix Button color on DarkTheme.
- [ ] feat: new features, e.g. add Snackbar component.
- [ ] refactor: code refactor, e.g. new folder structure for components.
- [ ] docs: changes into documentation, e.g. add usage example for Button.
- [ ] test: adding or updating tests, eg unit, snapshot testing.
- [x] chore: tooling changes, e.g. change circleci config.
- [x] BREAKING CHANGE: for changes that break existing usage, e.g. change API of a component.

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)

## NPM Audit
- **Before** this PR:
	- found 128 vulnerabilities (16 low, 53 moderate, 59 high) in 2455 scanned packages
- **After** this PR:
	- found 78 vulnerabilities (13 low, 8 moderate, 57 high) in 2603 scanned packages

### CODE REVIEW

File: ios/rwallet/Info.plist [hunk 1]
```diff
@@ -30,8 +30,6 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
-		<key>NSAllowsArbitraryLoads</key>
```
Comment (martinmedina) [inline ios/rwallet/Info.plist:33]: good catch!

File: patches/parse+3.2.0.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/parse/lib/react-native/EventEmitter.js b/node_modules/parse/lib/react-native/EventEmitter.js
```
Comment (martinmedina) [inline patches/parse+3.2.0.patch:1]: I don't think this file should be included

**Review by martinmedina — APPROVED** [review]
The PR looks fine so here is my approval 👍🏼

File: patches/parse+3.2.0.patch (hunk 2 above)
Comment (martinmedina) [reply patches/parse+3.2.0.patch:1]: It should, ignore this comment!

Comment (patogallaiovlabs) [reply patches/parse+3.2.0.patch:1]: Its a known issue, it is this patch:
https://github.com/microsoft/react-native-windows/issues/7451#issuecomment-856392511

**Comment by lucachaco** [comment]
tRBTC sent from IOS
image

**Pull request merged**